Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -73,10 +73,12 @@ public struct WorldScaleGeoTrackingSceneView: View {
         
         if ARGeoTrackingConfiguration.isSupported {
             configuration = ARGeoTrackingConfiguration()
+            _goal = .init(initialValue: .geoTracking)
         } else {
             configuration = ARWorldTrackingConfiguration()
-            configuration.worldAlignment = .gravityAndHeading
+            _goal = .init(initialValue: .tracking)
         }
+        configuration.worldAlignment = .gravityAndHeading
```

Comment:
According to my test, the default `worldAlignment` for `ARGeoTrackingConfiguration` is `gravityAndHeading`, and for `ARWorldTrackingConfiguration` is `gravity`.

Nonetheless, I  moved this line here so it is clearer for both configurations, we set it to `gravityAndHeading`

Reply:
After some more thoughts, I'll revert this change